package.json

```diff
@@ -41,7 +41,7 @@
     "start:docker:embed": "cd libs/embed && npm run start:docker",
     "start:docker:widget": "cross-env nx run-many --target=start:docker --projects=@novu/widget",
     "docker:build": "pnpm -r --if-present --parallel docker:build",
-    "build": "nx run-many --target=build --exclude=angular-notification-center-example,vue-notification-center-example --all",
```

Comment:
Was this removed on purpose? What is behind this?

Reply:
I recently removed these examples in this [PR](https://github.com/novuhq/novu/pull/3232) and moved them to our new repo novuhq/examples in this [PR](https://github.com/novuhq/examples/pull/1)